### PR TITLE
Lab3...

### PR DESCRIPTION
**Pull request by afonyaa**

### CODE REVIEW

**Comment by Maxkoz777** [comment]
Please, add information for tests into PYTHON.md

File: .github/workflows/pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      - uses: actions/checkout@v3
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install flake8 pytest
+          if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
```
Comment (AKurmazov) [inline .github/workflows/pipeline.yml:26]: nit: There is an action that caches the dependencies (boosted x2.5 my builds). Might be worth looking into it

**Comment by none-word** [comment]
You are a fast learner. 
Also, please, fix comments above

**Comment by Homa3030** [comment]
👍🏻

**Pull request closed without merging**